converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java

```diff
@@ -411,11 +405,24 @@ private void createLookupTable(final Element maps, final Component transformatio
 		}
 	}
 
-	private void createParameters(final Map<String, String> parameterMappings, final Element component) {
+	private void createParameters(final Component component, final Element componentElement) {
 
 		// TODO: parse parameter values that can be simple string values, JSON objects or JSON arrays (?)
 		// => for now we expect only simple string values
 
+		final String funtionName = component.getFunction().getName();
+
+		if (LOOKUP_FUNCTIONS.contains(funtionName)) {
+
+			final String lookupNameAttr = funtionName.equals(METAMORPH_FUNCTION_LOOKUP) ? "in" : "map";
```

Comment:
"in" and "map" could also be a constant here, or? (i.e. private static final ...)